tests/Install-Scripts.Test/GivenThatIWantToGetTheSdkLinksFromAScript.cs

```diff
@@ -8,11 +8,14 @@
 using System.Collections.Generic;
 using Microsoft.NET.TestFramework.Assertions;
 using FluentAssertions;
+using System.Text.RegularExpressions;
 
 namespace Microsoft.DotNet.InstallationScript.Tests
 {
     public class GivenThatIWantToGetTheSdkLinksFromAScript : TestBase
     {
+        private const string _feedCredential = "478a920c-2217-49f2-9c31-2fc3b4fef7cb";
```

Comment:
please add comments describing the credential.  valid credentials should not be enlisted.  No problem if this is a string, without any access or authorization, merely used for testing.